app/models/ad.rb

```diff
@@ -1,4 +1,6 @@
 class Ad < ApplicationRecord
+  KINDS = %w[Smještaj Prijevoz Prijava\ štete Usluge\ popravke]
```

Comment:
"Usluge popravke" sounds a bit weird?

Reply:
It does, but I'm not sure how to better name it... Popravci maybe? But then it could be interpreted as need, and not services... Not sure

Reply:
"Popravci i usluge" maybe?

Reply:
@vfonic Sounds good! We tabled 'Prijava štete' for now. Something we can work on later.